### DISCUSSION TITLE
Mirabella Genio Motion Sensor

### DISCUSSION THREAD

**Discussion by sambarlick** [make-all/tuya-local#1527] [Q&A]
Hi,

I have a couple of the Mirabella Genio Motion Sensors (https://github.com/sambarlick/tuya-local/blob/main/custom_components/tuya_local/devices/mirabella_genio_motion_sensor.yaml).

The product id in this yaml file is the same as my devices, but I can't connect them to tuya local.
The integration just gives the standard error "Unable to connect to your device with those details. It could be an intermittent issue, or they may be incorrect." I have made sure the Device id, local key and IP address are all correct. I have tried every protocol version (even auto). I have even made sure the device is awake and being registered on my router.

Is there something I'm doing wrong or missing?

Thanks,
Sam

![Screenshot_20240115-202213](https://github.com/make-all/tuya-local/assets/47462284/7e9b2eea-e5c6-463a-a901-751cd115ea96)

**Comment by make-all** [comment]
The link above is to a fork of this repository. Is the file you linked here?

**Reply by sambarlick** [reply]
Sorry, yes it is, heres the link...https://github.com/make-all/tuya-local/blob/main/custom_components/tuya_local/devices/mirabella_genio_motion_sensor.yaml
